site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -161,6 +161,22 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#{base_field_name}.externalIdentifierConfiguration.emailVerificationIdTimeToLiveInSeconds# [type]#[Integer]# [since]#Available since 1.8.0#::
 The time in seconds until an email verification Id is no longer valid and cannot be used by the Verify Email API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator.length# [type]#[Integer]# [since]#Available since 1.27.0#::
```

Comment:
Use `{base_field_name}` instead of `tenant` because this file is used for both single and multiple responses.